.github/release-drafter.yml

```diff
@@ -1,7 +1,10 @@
 template: |
   ## Release Notes
 
-  
+
+  ## Breaking Changes
```

Comment:
Is an admin planning to sort through all of the changes and make this determination ? I think that will be harder to decide on than something like "fixes" vs "new features". Do we have recommended criteria, here ?

Reply:
@emdupre we did not come up with a set of criteria, but the thought was that things like renaming the log file would go here, since it would break an exiting workflow.